libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -787,7 +787,7 @@ bool NavEKF3::InitialiseFilter(void)
     }
 #endif
 
-    if (core == nullptr) {
+    if (!core) {
```

Comment:
`core == nullptr` is our current standard.

This patch moves us away from that standard.